sdk/core/azure-core-test/src/main/java/com/azure/core/test/TestBase.java

```diff
@@ -53,6 +53,11 @@ public abstract class TestBase implements BeforeEachCallback {
      */
     public static final String AZURE_TEST_SERVICE_VERSIONS_VALUE_ALL = "ALL";
 
+    /**
+     * Specifies that the out of process test proxy should be used.
+     */
+    protected static boolean enableTestProxy;
```

Comment:
Since it's static that gets somewhat awkward (as the derived class would have to refer to the setter with the superclass name.) I don't mind that but thought I'd point it out. What do you think?